src/webui/src/components/trial-detail/Para.tsx

```diff
@@ -213,6 +214,10 @@ class Para extends React.Component<ParaProps, ParaState> {
 
             }
         }
+        parallelAxis.push({
+            dim: i,
+            name: 'default metric'
+        });
```

Comment:
can we show other metrics effect (corresponding to different params)?

Reply:
I'll create issue to follow up this.